### PR TITLE
tui: Available help system

### PR DESCRIPTION
**Pull request by poncovka**
The help system is now available from tui. Spokes with a help to
display have an extra option 'h' in their prompt.

### CODE REVIEW

File: pyanaconda/ui/tui/hubs/__init__.py [hunk 1]
```diff
@@ -113,6 +114,12 @@ def input(self, args, key):
                     if not spoke.completed and spoke.mandatory:
                         print(_("Please complete all spokes before continuing"))
                         return False
+            # TRANSLATORS: 'h' to help
+            elif key == C_('TUI|Spoke Navigation', 'h'):
+                if self.has_help:
+                    help_path = ihelp.get_help_path(self.helpFile, self.instclass, True)
+                    self.app.switch_screen_modal(HelpScreen(self.app, help_path))
+                    return True
```
Comment (vpodzime) [inline pyanaconda/ui/tui/hubs/__init__.py]: I know there's just ``False`` above, but I believe there were some constants for ``input()`` method's return values which made more sense than ``True, False``. Could you please check that and change this and the new ``input`` method added below to use the constants?

File: pyanaconda/ui/tui/spokes/langsupport.py [hunk 2]
```diff
@@ -122,7 +123,7 @@ def input(self, args, key):
             self.app.switch_screen(self, None)
             return True
         else:
-            return key
+            return NormalTUISpoke.input(self, args, key)
```
Comment (jkonecny12) [inline pyanaconda/ui/tui/spokes/langsupport.py]: Isn't there a way how to solve this globally? I mean in some base class or similar place.

File: pyanaconda/ui/tui/hubs/__init__.py (hunk 1 above)
Comment (poncovka) [inline pyanaconda/ui/tui/hubs/__init__.py]: Fixed.

File: pyanaconda/ui/tui/spokes/langsupport.py (hunk 2 above)
Comment (poncovka) [inline pyanaconda/ui/tui/spokes/langsupport.py]: Fixed.

**Pull request merged**